### PR TITLE
Add asciidoctor Task

### PR DESCRIPTION
**Pull request by jandradap**
Asciidoctor text processor task

https://asciidoctor.org

# Changes
This PR introduces tasks for Asciidoctor.

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [X] Follows the [authoring recommendations](https://github.com/tektoncd/catalog/blob/main/recommendations.md)
- [X] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [X] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality of task changed or new task added)
- [X] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)
- [X] Complies with [Catalog Organization TEP][TEP], see [example]. **Note** [An issue has been filed to automate this validation][validation]
  - [X] File path follows  `<kind>/<name>/<version>/name.yaml`
  - [X] Has `README.md` at `<kind>/<name>/<version>/README.md`
  - [X] Has mandatory `metadata.labels` - `app.kubernetes.io/version` the same as the `<version>` of the resource
  - [X] Has mandatory `metadata.annotations` `tekton.dev/pipelines.minVersion`
  - [X] mandatory `spec.description` follows the convention

          ```

          spec:
            description: >-
              one line summary of the resource

              Paragraph(s) to describe the resource.
          ```

_See [the contribution guide](https://github.com/tektoncd/catalog/blob/master/CONTRIBUTING.md)
for more details._

---

[TEP]: https://github.com/tektoncd/community/blob/master/teps/0003-tekton-catalog-organization.md
[example]: https://github.com/tektoncd/catalog/tree/master/task/git-clone/0.1
[validation]:  https://github.com/tektoncd/catalog/issues/413

### CODE REVIEW

**Comment by jandradap** [comment]
hi @vinamra28 , is there a way to solve the PRs faster? Thx

File: task/asciidoctor/0.1/README.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+*** table with some cell formatting
+**** sequential paragraphs
+***** admonition blocks, but use them sparingly
+*** bullet list with nesting
+** numbered list with nesting
+** definition list
+*** sidebar
+```
+
+**Ouput:**
```
Comment (vinamra28) [inline task/asciidoctor/0.1/README.md:92]: nit (feel free to ignore this if there are no further changes required)
```suggestion
**Output:**
```

File: task/asciidoctor/0.1/README.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+Install git-clone task from catalog
+```
+https://raw.githubusercontent.com/tektoncd/catalog/main/task/git-clone/0.3/git-clone.yaml
+```
+
+## Workspaces
+* **source** : A Workspace containing your source directory.
+
+## Parameters
+* **ASCIIDOC_CMD** : Command to convert AsciiDoc files asciidoctor, asciidoctor-pdf, asciidoctor-epub3, etc. Default ```asciidoctor-pdf```.
```
Comment (vinamra28) [inline task/asciidoctor/0.1/README.md:24]: can you please list all the possible commands or add a link where a user can find that?

**Review by vinamra28 — COMMENTED** [review]
overall PR looks good :+1:. Can you please address the following comments?

File: task/asciidoctor/0.1/README.md (hunk 1 above)
Comment (jandradap) [reply task/asciidoctor/0.1/README.md:92]: Done

File: task/asciidoctor/0.1/README.md (hunk 2 above)
Comment (jandradap) [reply task/asciidoctor/0.1/README.md:24]: Done

**Review by vdemeester — APPROVED** [review]
/lgtm

**Pull request merged**